### ISSUE TITLE
Blue screen of death on launch

### ISSUE THREAD

**Issue by mundodisco8** [Aloshi/EmulationStation#551]
Hi!
I'm having a BSOD every time I run ES. The type of BSOD I have is a "PAGE_FAULT_IN_NONPAGED_AREA" and it's caused by ddkmd.sys. The only results google throws are related to a company called OSBASE, that makes Windows display drivers. I thought that I didn't have any of their products installed, but one of them is called iDisplay, and I linked that with Duet (an app I installed to use my iPad as a second screen). I tried to uninstall Duet, and ES works. Reinstalling Duet causes the BSODs again.

I have a Vaio laptop running Windows 10 with an ATI HD4650 with the latest drivers (which are legacy W8 ones). I will chime the Duet guys too, as they are having some issues with w10.

Also, and probably not related, when Duet is not present and I am able to run ES, I have a serious issue of screen tearing, that is fixed by disabling Vsync. I don't really need Vsync in ES, as it is working in Retroarch and I don't have the infinite fps issue, but I leave it here just in case they are related. If you think they are not, I can open another ticket.

**Comment by XombyCraft** [comment]
Do any other programs that open in "full screen" mode cause a similar issue?
I've had funky things like that happen on multi-display setups(NOT using duet, just legit multi-screens) when a program isn't designed to properly take into consideration multiple displays, or for whatever reason chooses to be cranky...  
[I'm not affiliated with ES in any way, but I am an IT guy]

**Comment by mundodisco8** [comment]
Hi! Thanks for taking the time to share your experience.

I forgot to mention, but with Duet uninstalled, I tried to run it in windowed mode and it fixed the screen tearing with vsync on. After your post, I tried to run in windowed mode, and it ran perfectly, no screen tearing issues with vsync on. Searching on google I found a post in steam of a guy that had the same issue running the Binding of Isaac, where he would get the same BSOD message and he was able to fix it by uninstalling duet. I tried to run it and it worked (at least in windowed mode), maybe it would crash if I try to run it in fullscreen mode.

So that works for me. I will run it in windowed mode. If somebody has this issue, maybe this could help them. In the meantime, I will leave this open and wait for the developer to decide if it will be fixed or not.
